changelog/add-5880-track-overview-balance

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
```

Comment:
This is `dev` – not adding anything that benefits merchants.

Reply:
Good point. Addressed [54c7942](https://github.com/Automattic/woocommerce-payments/pull/6307/commits/54c79428d719758ead0f4983884281c834d67010).